specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,6232 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Machine Learning Services",
```

Comment:
why the title changed from "Azure Machine Learning Workspaces" to "Azure Machine Learning Services" ?